src/core/prototypes/plot_segments/virtualScroll.tsx

```diff
@@ -145,20 +143,27 @@ export const VirtualScrollBar: React.FC<VirtualScrollBarPropertes> = ({
       if (newPosition > 100) {
         newPosition = 100;
       }
-      if (newPosition - widthPerBarPercent * 100 < 0) {
-        newPosition = 0;
+      if (dataType == AxisDataBindingType.Categorical) {
+        if (newPosition - widthPerBarPercent * 100 < 0) {
```

Comment:
nit: if the variable needs to be multiplied by 100, it isn't a percent, it's a ratio. consider variable name change.